### PR TITLE
Removes experimental.h, moving remaining declarations to ts.h

### PR DESCRIPTION
**Pull request by zwoop**

### CODE REVIEW

File: doc/developer-guide/api/functions/TSIpStringToAddr.en.rst [hunk 1]
```diff
@@ -53,4 +49,3 @@ Notes
 =====
 
 This API may be changed in the future version since it is experimental.
-
```
Comment (ywkaras) [inline doc/developer-guide/api/functions/TSIpStringToAddr.en.rst:56]: Removing this unnecessary blank line seems unrelated to this PR, should be in it's own PR.

File: src/api/InkAPI.cc [hunk 2]
```diff
@@ -6575,10 +6575,6 @@ tsapi::c::TSPluginVCIOBufferWaterMarkGet(TSHttpTxn txnp)
   return TS_IOBUFFER_WATER_MARK_PLUGIN_VC_DEFAULT;
 }
 
-// The API below require timer values as TSHRTime parameters
-// which are in nanoseconds. Use the TS_HRTIME macros defined
```
Comment (ywkaras) [inline src/api/InkAPI.cc:6579]: Isn't it useful to know the parameters are in nanoseconds?

Comment (zwoop) [reply src/api/InkAPI.cc:6579]: No it’s in the docs

File: doc/developer-guide/api/functions/TSIpStringToAddr.en.rst (hunk 1 above)
Comment (zwoop) [reply doc/developer-guide/api/functions/TSIpStringToAddr.en.rst:56]: Brutal

Comment (ywkaras) [reply doc/developer-guide/api/functions/TSIpStringToAddr.en.rst:56]: Welcome to my world.

**Review by ywkaras — APPROVED** [review]
